website/docs/components/link/inline/partials/guidelines/guidelines.md

```diff
@@ -1,3 +1,101 @@
 ## Usage
 
-Usage documentation for this component is coming soon. In the meantime, help us improve this documentation by letting us know how your team is using it in [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS) (internal only).
\ No newline at end of file
+### When to use
+
+- To navigate to a new destination, either internally or externally.
+- Within a block of body text or other text element.
+
+### When not to use
+
+- As a standalone element, consider [Standalone Link](/components/link/standalone).
+- To trigger an action or event, like a form submission, consider [Button](/components/button).
+
+## Usage in Figma
+
+The Inline Link is only published as a component in Ember, there is no corresponding component in Figma. To achieve the same results, apply **text** and **color** styles from the [Foundations library](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=2130%3A2&t=6XBReWOxMRTiW1Iv-1) to your text.
+
+The Ember component inherits the styles of the text around it, ensure that the font weight defined in Figma is the same as the body text it is contained within.
+
+- Text styles can be found under text styles `Body` or `Code`.
+- Color styles:
+    - `Components / Link inline / Foreground / Primary`
+    - `Components / Link inline / Foreground / Secondary`
+
+!!! Info
+
+In Figma we publish links with a font-weight of `Medium`. Depending on the surrounding text the link style may need to be overriden to match.
+!!!
+
+## Color
+
+### Primary
+
+We recommend using the `primary` variant as the default and for more important links.
+
+#### Primary example
+
+<span class="hds-typography-body-300">Lorem ipsum <Hds::Link::Inline @color="primary" @href="...">dolor</Hds::Link::Inline> sit amet, consectetur adipiscing elit.</span>
+
+### Secondary
+
+Use the secondary variant for less important links, when the primary link can’t be used, or when there are multiple links in a block of text.
```

Comment:
> when there are multiple links in a block of text.

This part may get confusing for consumers. What would we expect a consumer to do if one paragraph has only 1 link and the next paragraph has multiple links? It would be a little strange to have a mix-and-match of link styles.

Reply:
I've added some clarifying content around this topic.